### ISSUE TITLE
Single lesion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10150]
**Preferred term label:**
Single lesion

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0577304

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Granuloma faciale

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Not sure if this is meant to refer to "Unifocal"? (Which would need to be a new term)

**Comment by MickeySegal** [comment]
I would use unifocal to describe an EEG, but single lesion to describe a tumor, skin eruption. or infection as in the references at https://www.ncbi.nlm.nih.gov/medgen/C0577304

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000914
- label: Single lesion
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality whose distribution and appearance are solitary and localized to a part of the skin, a tissue or an organ that differs from the surrounding healthy tissue.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10150
- synonyms:

**Comment by MickeySegal** [comment]
But HP:6000914 was assigned to a different concept in https://github.com/obophenotype/human-phenotype-ontology/issues/8865
